examples/importation.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
```

Comment:
I like the idea of having an example, but I feel that this will get out of date as soon as we start updating the importer. We probably should move this into a test case, and point the users there in the documentation.